openhtf/io/frontend/__main__.py

```diff
@@ -86,6 +87,31 @@ class Station(mutablerecords.Record(  # pylint: disable=too-few-public-methods,n
       framework: A dictionary representation of a TestExecutor object.
       test: A dictionary representation of a TestState object.
     """
+  def Refresh(self):
+    """Update state with a GET request to OpenHTF's HTTP API."""
+    try:
+      response = requests.get('http://%s:%s' % self.hostport)
+      if response.status_code == 200:
+        state = json.loads(response.text)
+        self.station_id = state['framework']['station_id']
```

Comment:
probably worth logging something if `self.station_id is not UNKNOWN_STATION_ID and self.station_id != state['framework']['station_id']`.